Resources/Prototypes/Catalog/VendingMachines/Inventories/happyhonk.yml

```diff
@@ -0,0 +1,9 @@
+﻿- type: vendingMachineInventory
+  id: HappyHonkDispenserInventory
+  startingInventory:
+    HappyHonkRandom: 15
+    HappyHonkMime: 5
```

Comment:
This is way too much, it should be 1/4 of this at max, see https://github.com/space-wizards/space-station-14/issues/12462